### ISSUE TITLE
DLMM.create is not a function

### ISSUE THREAD

**Issue by keyboard-warlord** [MeteoraAg/dlmm-sdk#125]
Hello!

I recently installed this library as a dependency for my project, but I encountered an issue right at the start. I'm trying to connect to a specific pool following the steps outlined in the documentation: [DLMM SDK Integration Guide](https://docs.meteora.ag/dlmm/dlmm-integration/dlmm-sdk#install).

Here is my code:

```
import DLMM, { StrategyType } from "@meteora-ag/dlmm";

const dlmmPool = await DLMM.create(alchemyApi, details.poolAddress);
```

However, when I start the project, I get the following error:

```
const dlmmPool = await DLMM.create(alchemyApi, details.poolAddress);
                                ^

TypeError: DLMM.create is not a function
```

Could you please clarify how I can resolve this issue?

For context:

Type definitions are being imported and recognized correctly.
If you need any additional information, feel free to let me know.
Thank you!

**Comment by professorr-x** [comment]
this works for me:
`const dlmmPool = await DLMM.default.create(alchemyApi, details.poolAddress);`

**Comment by Magnuskeel** [comment]
I have exactly the same issue unfortunately. I think it might have to do with the tsconfig(module, target settings),  but honestly tried everything at this point.... like the OP I am able to import and enter to debug the code but the static create method returns as undefined.

**Comment by keyboard-warlord** [comment]
> I have exactly the same issue unfortunately. I think it might have to do with the tsconfig(module, target settings), but honestly tried everything at this point.... like the OP I am able to import and enter to debug the code but the static create method returns as undefined.

Yes, I also concluded that the issue is likely with my environment setup. However, I'm using meteora-sdk in only a very small part of my project. So, I decided to work around it by simply copying the repository and using DLMM directly from the source files, without building it.

**Comment by Magnuskeel** [comment]
After some trial and error I came up with a small casting hack given our environments ;)

The import statement remains as documented but you need to access the "default" property of what seems to the the DLMM object even though the compiler wont let you.

const dlmm = ((DLMM)as any).default;
const dlmmPool = await dlmm.create(connection, new PublicKey(METEORA_POOL));

This should work.

**Comment by ddevejian** [comment]
I am also experiencing this issue.

Interestingly, with the import statement as:
`import DLMM from "meteora-ag/dlmm" `
I get the DLMM.create is not a function error.  Also, my IDE's syntax checker does not recognize it as valid syntax.  Even after the `const dlmm = ((DLM as any).default` hack, the syntax checker does not know that dlmm has functions.

If on the other hand I change the import statement to:
`import {DLMM} from "meteora-ag/dlmm"`
I get an error "The requested module '@meteora-ag/dlmm' does not provide an export named 'DLMM'" although I can see the export in the code.  But my IDE's syntax checker does seem do see DLMM as a valid class and sees all the associated functions.  So my syntax checker recognizes the export, but the compiler doesnt?

**Comment by CryptoUncleSam** [comment]
> this works for me: `const dlmmPool = await DLMM.default.create(alchemyApi, details.poolAddress);`

This work perfect for me

**Comment by hiendv** [comment]
It's because of their build config (tsup, exports, etc). It doesn't work with `"type": "module"`.

This is another fix:
```ts
import { createRequire } from "module";
const require = createRequire(import.meta.url);
const DLMM = require("@meteora-ag/dlmm").default;
```
